crunchbase/README.md

```diff
@@ -0,0 +1,63 @@
+# Crunchbase Connector
+
+This package is a utility for connecting Cohere to Crunchbase.
+
+## Configuration
+
+The search connector requires the following environment variables:
+
+```
+CRUNCHBASE_API_KEY
+```
+
+This variable should contain the API key of the Crunchbase account. To get the API key, use the
+instructions [here](https://www.crunchbase.com/account/integrations/crunchbase-api)
+
+```
+CRUNCHBASE_CONNECTOR_API_KEY
+```
+
+This variable is used to authenticate users using bearer token.
+
+### Optional configuration
+
+```
+CRUNCHBASE_SEARCH_LIMIT
+```
+
+This variable may contain the maximum number of results to return from Crunchbase. Default value is 20.
+
+## Limitations
```

Comment:
Very small nit to just move this section to right before the Configurations one, thank you for adding this!

Reply:
done! feel free to merge it @tianjing-li 👍